.github/workflows/cmake.yml

```diff
@@ -0,0 +1,29 @@
+# Note: CMake support is community-based. The maintainers do not use CMake
+# internally.
+
+name: cmake
+
+on:
+    push:
+    pull_request:
+        branches:
+            - main
+
+permissions:
+    contents: read
+
+jobs:
+  cmake_min:
```

Comment:
Can we make this name consistent with other job names, or make them consistent with it?